models/renalStructures_UNEST_segmentation/configs/metadata.json

```diff
@@ -3,7 +3,8 @@
     "version": "0.1.0",
```

Comment:
Hi @tangy5 , here the version number should match the changelog

Reply:
The version indeed cause confusion, let's remove it for now. If there are other versions in the future, will keep log change.

Reply:
No, "version" here should represent the latest version, in your case is "0.1.2"